tests/Http/Controllers/ResourceControllerTest.php

```diff
@@ -445,6 +445,63 @@ public function can_edit_resource_if_resource_is_read_only()
             ->assertSee($post->body);
     }
 
+    /**
+     * @test
+     * https://github.com/duncanmcclean/runway/pull/370
+     */
+    public function can_edit_resource_with_nested_field_cast_to_object_in_model()
```

Comment:
Yeah, weird...not actually sure how that `mergeCasts` worked when I initially set this up; test was very much built to fail initially, but now I can't replicate it (which actually makes sense, I think, since the merged casts are presumably not persisted on the test `get` request). Spent some time on that this morning and tried to keep the modified version as light as possible, but let me know if the new column/field is a bridge too far. Patched in incoming commits.